packet.go

```diff
@@ -106,10 +106,10 @@ type packetHeader struct {
 	// Consult getHeaderType if you want a value that corresponds to the
 	// definition of packetType.
 	Type                    packetType
+	Version                 VersionNumber
```

Comment:
Why move this?

Reply:
Things are serialized in the order they appear in the data structure and VN is the second value. See: 

https://tools.ietf.org/html/draft-ietf-quic-transport-11#section-4.1